SampleApp/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,10 @@
+#Mon Jan 20 13:25:50 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+<<<<<<< HEAD
+=======
 distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+>>>>>>> 5279c060dbe3b84cf1a831cc2b7fba94da5734e3
```

Comment:
This line has to be removed after resolving conflicts. It is to denote conflict.